agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/exporter/Exporter.java

```diff
@@ -822,6 +822,22 @@ private void exportRequest(SpanData span) {
     exportEvents(span, operationName, samplingPercentage);
   }
 
+  private static boolean getSuccess(SpanData span) {
+    if (span.getStatus().getStatusCode() == StatusCode.ERROR) {
+      return false;
+    }
+    if (span.getStatus().getStatusCode() == StatusCode.OK) {
+      // auto-instrumentation never sets OK, so this is explicit user override
+      return true;
+    }
+    Long statusCode = span.getAttributes().get(SemanticAttributes.HTTP_STATUS_CODE);
+    if (statusCode == null) {
+      return true;
+    }
+    // override default OpenTelemetry mapping of status codes 4xx
```

Comment:
what is the default opentelemetry status code mapping? what about redirect error code that is in the 300 range?

Reply:
the default otel mapping is
* >= 500 --> `ERROR`
* otherwise --> `UNSET`

prior to 1.9.0, the default otel mapping was
* >= 400 --> `ERROR`
* otherwise --> `UNSET`